src/librustdoc/html/render.rs

```diff
@@ -964,10 +967,11 @@ themePicker.onblur = handleThemeButtonsBlur;
                     variables.push(line.clone());
                     continue;
                 }
-                if !line.starts_with(key) {
+                if !line.starts_with(key) || line.starts_with(&format!(r#"{}["{}"]"#, key, krate)) {
                     continue;
                 }
-                if line.starts_with(&format!(r#"{}["{}"]"#, key, krate)) {
+                if line.starts_with(MINIFIED_VARIABLES) {
+                    is_minified = true;
```

Comment:
This is dead code. The above:

```rust
if !line.starts_with(key) || line.starts_with(&format!(r#"{}["{}"]"#, key, krate)) {
    continue;
}
```

means that `line.starts_with(MINIFIED_VARIABLES)` can never be true. This means that the minification variables are missing if the last call to rustdoc used `--disable-minification` but previous ones didn't.